releases/v0.12.md

```diff
@@ -0,0 +1,74 @@
+# v0.12
+
+This document contains all release notes pertaining to the `v0.12.x` releases of Vela.
+
+## For Vela Administrators
+
+* When upgrading from `v0.11`, please take note of the [migration information](/migrations/v0.12/README.md)
+
+## Full release notes available on [github.com/go-vela](https://github.com/go-vela)
+
+* [go-vela/ui](https://github.com/go-vela/ui/releases)
+* [go-vela/server](https://github.com/go-vela/server/releases)
+* [go-vela/worker](https://github.com/go-vela/worker/releases)
+* [go-vela/cli](https://github.com/go-vela/cli/releases)
+* [go-vela/sdk-go](https://github.com/go-vela/sdk-go/releases)
+* [go-vela/types](https://github.com/go-vela/types/releases)
+
+## v0.12.0
+
+Happy February Vela Community!
+
+Below is a summary of what's new in 0.12.0
+
+### Breaking Changes
+
+* (types) fix(secret)!: remove implicit repo secrets and clean up secrets testing by @GregoryDosh in [#210](https://github.com/go-vela/types/pull/210)
+  - existing implicit secrets usage will cease to work
```

Comment:
Can you add some more clarification on the implications of this? Bit unclear to me what this means without reading through the source code in that pull request.

Reply:
added a commit to check out - does that help?

Reply:
I thought this change was reverted and not introduced into `v0.12.x`?

Revert PR - https://github.com/go-vela/types/pull/229

Reimplemented PR - https://github.com/go-vela/types/pull/230

Reply:
Very late to the party but it probably woulda been nice to add what this [character](https://github.com/go-vela/types/pull/210/files#diff-4df2bf19e26362114b475b25b13f8ffd4b2dc54037729f3f28a7185c92caaabcR25) was, I had to google and do some reading on it. Woulda helped know from the PR link the purpose in the release notes.

CC: @GregoryDosh

Reply:
@jbrockopp oh yes.. my fault. i removed some of the PRs and this one slipped through! when i pulled the release notes 😵

Reply:
It's all good! ..for a moment there I thought I was going crazy 😅 🤪

Reply:
fixed - nothing to see on that topic for this release